<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>836</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="queryButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>38</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yuebaoButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>生成月报</string>
    </property>
   </widget>
   <widget class="QPushButton" name="excelButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>153</y>
      <width>94</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>导出至Excel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>显示筛选项：</string>
    </property>
    <property name="buddy">
     <cstring>allBox</cstring>
    </property>
   </widget>
   <widget class="QCheckBox" name="allBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>全部</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="buoyBox">
    <property name="geometry">
     <rect>
      <x>1014</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>浮标状态</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="watertBox">
    <property name="geometry">
     <rect>
      <x>907</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>水温</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="waveBox">
    <property name="geometry">
     <rect>
      <x>318</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>波浪</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="humBox">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>温湿度</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="curBox">
    <property name="geometry">
     <rect>
      <x>1121</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>海流和水质</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="airpBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>气压</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="windBox">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>风</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>227</y>
      <width>1201</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1201</width>
      <height>501</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>1000</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>250</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ContiguousSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>125</number>
    </attribute>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>20</y>
      <width>370</width>
      <height>170</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>370</width>
      <height>170</height>
     </size>
    </property>
    <property name="title">
     <string>查询条件</string>
    </property>
    <widget class="QDateEdit" name="dateEditStart">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2017</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>数据时间：</string>
     </property>
     <property name="buddy">
      <cstring>dateEditStart</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>至</string>
     </property>
     <property name="buddy">
      <cstring>dateEditEnd</cstring>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEditEnd">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2018</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="date">
      <date>
       <year>2018</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>浮标号：</string>
     </property>
     <property name="buddy">
      <cstring>comboBox</cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>87</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>701</width>
      <height>170</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>接受率统计数据</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>应接收数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_yjss">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>90</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CDMA接收数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cjs">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>90</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> 总接收数:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_zj">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>90</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>北斗接收数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_bdjs">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>130</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>北斗实时接收率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_bssjsl">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>  总实时接收率:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_zssjsl">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CDMA实时接收率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cssjsl">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>  总实时接收数:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_zssjs">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CDMA实时接收数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cssjs">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>80</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>北斗实时接收数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nssjs">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>yuebaoButton</tabstop>
  <tabstop>excelButton</tabstop>
  <tabstop>allBox</tabstop>
  <tabstop>waveBox</tabstop>
  <tabstop>windBox</tabstop>
  <tabstop>humBox</tabstop>
  <tabstop>airpBox</tabstop>
  <tabstop>watertBox</tabstop>
  <tabstop>dateEditStart</tabstop>
  <tabstop>dateEditEnd</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>queryButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
